charts/minecraft/values.yaml

```diff
@@ -3,6 +3,8 @@ image: itzg/minecraft-server
 imageTag: latest
 imagePullSecret: ""
 
+replicaCount: 1
```

Comment:
It feels like this needs a big warning similar to what you described at the top of the PR.

Reply:
you think a comment in the values.yaml is enough? ill add one

Reply:
there we go

Reply:
Thanks!